app/models/user.rb

```diff
@@ -2,14 +2,14 @@ class User < ActiveRecord::Base
   include PgSearchCommon
 
   has_attached_file :avatar,
-    styles: { medium: "300x300>", thumb: "100x100>#", head: "32x32#" },
+    styles: { medium: '300x300>', thumb: '100x100>', head: '32x32#'},
     default_url: 'standard_:style.png'
 
   before_validation :clear_avatar
 
   validates_attachment_size :avatar, less_than: 1.megabyte
   validates_attachment_content_type :avatar,
-    content_type: ['image/jpeg', 'image/png', 'image/gif']
+    content_type: %r/\Aimage\/.*\Z/
```

Comment:
I just took it from the paperclip manual :P May as well revert it back to jpeg/png/gif, we shouldn't get other pictures anyway